pom.xml

```diff
@@ -2,14 +2,15 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.batkinson.plugins</groupId>
+    <groupId>com.github.pixeon.plugins</groupId>

```

Comment:
Did you inadvertently commit these changes to the same branch, forgetting that it would update the pull request? Changing the groupid doesn't make sense unless you are telling me not to publish to maven central, instead relying on your own repo instead. I can't accept the changes unless this is removed.